### DISCUSSION TITLE
Filter Multi select Chip Overflow horizontally

### DISCUSSION THREAD

**Discussion by sagrawal9689** [KevinVandy/material-react-table#242] [Q&A]
![Screenshot from 2022-12-09 15-47-04](https://user-images.githubusercontent.com/57897337/206681902-7b7b8c2a-f11e-4509-91f1-c25c0c2c9172.png)

Currently the overflow for multi select spans vertically which looks bad, is there a way we can make it to overflow horizontally. 
example (State column): https://codesandbox.io/s/filter-multi-select-chip-overflow-horizontally-siijv1?file=/src/TS.tsx

**Comment by mahesh-kedari** [comment]
There should be a flag to display a list with vertical or horizontal scroll. 
Currently its just growing vertically which may not be a desired outcome in all the cases.

**Reply by sagrawal9689** [reply]
@mahesh-kedari I can not find any such flag for doing so.

**Reply by KevinVandy** [reply]
There's no flag, but you can customize the CSS of the head cell or the filter textfield however you need with `muiTableHeadCellProps` and `muiTableHeadCellFilterTextFieldProps` column options or props
